CHANGELOG.md

```diff
@@ -5,6 +5,13 @@ on [Keep a CHANGELOG](http://keepachangelog.com/). This project adheres to
 
 ## [Un-released]
 
+### Added changed removed and sorted
```

Comment:
These change log lines should be under separate headings (eg "Added", "Changed" and "Fixed") - see the http://keepachangelog.com for guidance on headings.